### DISCUSSION TITLE
When will templates be updated to v3?

### DISCUSSION THREAD

**Discussion by grantspilsbury** [payloadcms/payload#7883] [Feature Requests & Ideas]
I love Payload but I'm hesitant to start a new project, using a template, knowing they are using v2. Is this founded or a dumb way to think about starting a new project using a v2 template?

**Comment by javierlinked** [comment]
Search before creating new discussions, please. 

https://github.com/payloadcms/payload/discussions/7627

https://github.com/payloadcms/payload/discussions/6792
